README.md

```diff
@@ -26,28 +32,54 @@ There is a large number of templates that are ready to use and are officially su
+| `@asyncapi/java-spring-template`              | Generates Java Spring service                                   | [click here](https://github.com/asyncapi/java-spring-template)              |
+| `@asyncapi/java-spring-cloud-stream-template` | Generates Java Spring Cloud Stream service                      | [click here](https://github.com/asyncapi/java-spring-cloud-stream-template) |
+| `@asyncapi/python-paho-template`              | Generates Python service that uses Paho library                 | [click here](https://github.com/asyncapi/python-paho-template)              |
+| `@asyncapi/html-template`                     | Generates HTML documentation site                               | [click here](https://github.com/asyncapi/html-template)                     |
+| `@asyncapi/markdown-template`                 | Generates documentation in Markdown file                        | [click here](https://github.com/asyncapi/markdown-template)                 |
+| `@asyncapi/ts-nats-template`                  | Generates TypeScript NATS client                                | [click here](https://github.com/asyncapi/ts-nats-template/)                 |
+| `@asyncapi/go-watermill-template`             | Generates Go client using Watermill                             | [click here](https://github.com/asyncapi/go-watermill-template)             |
+| `@asyncapi/dotnet-nats-template`              | Generates .NET C# client using NATS                             | [click here](https://github.com/asyncapi/dotnet-nats-template)              |
+| `@asyncapi/php-template`                      | Generates PHP client using RabbitMQ                             | [click here](https://github.com/asyncapi/php-template)                      |
+| `@asyncapi/dotnet-rabbitmq-template`          | Generates .NET C# client using RabbitMQ                         | [click here](https://github.com/asyncapi/dotnet-rabbitmq-template)          |
 
 <!-- TEMPLATES-LIST:END -->
 
 You can find above templates and the ones provided by the community in **[this list](https://github.com/search?q=topic%3Aasyncapi+topic%3Agenerator+topic%3Atemplate)**
 
+# Generator Filters
+
+This is a library with [generator](https://github.com/asyncapi/generator/) filters that are used in more than one template. Use these filters to not double your work.
+
+This library consists of:
+
+- Custom filters. Check out [API docs](docs/api.md) for complete list
+- Lodash-powered filters. For the list of all available filters check [official docs](https://lodash.com/docs/)
+
 ## Contributing
 
+To Add a major/minor/patch Type Changes:
+
+1.  Create a new markdown file in the `.changeset` directory. The filename should be descriptive of the change.
+2.  Add the following content to the file in this particular format:
+
+### ---
+
+##### "[package-name-1]": [type] (minor/major/patch)
+
+##### "[package-name-2]": [type]
```

Comment:
so I can add these 2 in one markdown document, and indicate that one package is fix and another is major and that would work, right?

can you also extend this paragraph with:
- link to some changeset docs
- explanation of the example, so what happens when we merge a PR with such example markdown document - what will show up in npm and what will show up in github releases section

Reply:
yes we can do that.

Reply:
please add this

also something wrong with current formatting of code sample - also notice wrong indentation

![Screenshot 2024-07-22 at 13 51 44](https://github.com/user-attachments/assets/6b427880-06e4-41c6-abca-69282ca3aef9)